### ISSUE TITLE
Accent shadow and accent border no longer works

### ISSUE THREAD

**Issue by kaumac** [cschroeter/park-ui#110]
Hey there! 

Prior to v0.25 my inputs were working nicely with accent shadow and border customizations, like this:
```
semanticTokens: {
  colors: {
    border: {
        accent: { value: { base: '{colors.trusty.9}', _dark: '{colors.trusty.3}' } }
      }
    },
  },
  shadows: {
    accent: {
      value: '0 0 0px 5px {colors.trusty.3}'
    }
  }
}
```

After v0.25 both shadows and borders stopped working. I can see from the release notes that border.accent is deprecated but I can't really figure out how to get my inputs to work the way they did before...

Here is a screenshot of how it looked on v0.24:
<img width="691" alt="Captura de Tela 2023-12-05 às 23 37 41" src="https://github.com/cschroeter/park-ui/assets/1170740/f6a6ce20-80e7-4e74-9aa2-acfbdd68a434">

**Comment by cschroeter** [comment]
@kaumac 

Thanks for bringing this to my attention. This change was necessary in order to allow other focus / outline colors beside the `accent` color. In example for invalid inputs.

```tsx
// after
_focus: {
  borderColor: 'colorPalette.default',
  boxShadow: '0 0 0 1px var(--colors-color-palette-default)',
}

// before
_focus: {
  borderColor: 'border.accent',
  boxShadow: 'accent',
},
```

To solve that problem your `trusty` color palette needs to export this:


```tsx
export const indigo = {
  1: { value: { base: '{colors.trusty.light.1}', _dark: '{colors.trusty.dark.1}' } },
  // ... 12

  default: { value: '{colors.trusty.9}' },
  emphasized: { value: '{colors.trusty.10}' },
  fg: { value: '{colors.white}' },
  text: { value: '{colors.trusty.a11}' },
}
```

Let me know if that solves your problem.

**Comment by kaumac** [comment]
Hey @cschroeter thanks for the reply!
Unfortunately that doesn't quite solve it. In particular with the input shadows on focus, before I only had to add this:
```shadows: {
    accent: {
      value: '0 0 0px 5px {colors.trusty.3}'
    }
  }
  ```
  
To my theme and it automatically add the shadow on focus. This no longer works.
I also tried changing `accent` to `emphasized` but that also doesn't work.

I think having a semantic token for the focused input shadow is very useful for most people
